### PR TITLE
v5: Simplify navbars by requiring containers

### PR DESCRIPTION
**Pull request by mdo**
Closes #25274 by @zalog.

Unsure if we want to go with this, but I believe @MartijnCuppens suggested it somewhere. It does simplify behavior of the navbar a bit, and it ensures horizontal alignment from @zalog's original reported issue.

Thoughts @twbs/css-review?

TODO:

- [x] Mention the change in migration page

### CODE REVIEW

File: scss/_variables.scss [hunk 1]
```diff
@@ -114,8 +114,8 @@ $escaped-characters: (
 
 $enable-caret:                                true !default;
 $enable-rounded:                              true !default;
-$enable-shadows:                              false !default;
-$enable-gradients:                            false !default;
+$enable-shadows:                              true !default;
+$enable-gradients:                            true !default;
```
Comment (MartijnCuppens) [inline scss/_variables.scss]: Could you revert this?

**Comment by MartijnCuppens** [comment]
I like this idea, was thinking about doing this myself one day

**Comment by XhmikosR** [comment]
Rebased and addressed @MartijnCuppens's comment.

**Comment by XhmikosR** [comment]
Shouldn't we mention this in migration?

**Comment by MartijnCuppens** [comment]
> Shouldn't we mention this in migration?

We should

**Review by MartijnCuppens — APPROVED** [review]

**Comment by zalog** [comment]
Love it! Great work guys

**Pull request merged**